appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts

```diff
@@ -40,6 +40,9 @@ describe('SETUP: ', () => {
     const newFingerprintsValue = CommonData.updateRecipientPublicKey.newFingerprints;
 
     await mockApi.withMockedApis(async () => {
+      const times = await driver.getDeviceTime();
+      console.log('times', times);
```

Comment:
Is it needed or was just for debug purpose?

Reply:
yes, i will remove it, sorry

Reply:
@sosnovsky can we skip  test run after this update?

Reply:
Why skip it? I think it should pass without any issues